### PR TITLE
Update appraise defaults

### PR DESCRIPTION
**Pull request by AroneyS**
- [x] Update appraise `--imperfect` default to Species-level (0.95)
- [x] Log sequence identity used

### CODE REVIEW

**Comment by AroneyS** [comment]
Summarise also uses GENUS_LEVEL_AVERAGE_IDENTITY for sequence clustering. Should that change too? @wwood

**Comment by wwood** [comment]
Thanks @AroneyS can you also add a comment line (i.e. starts with #) giving the %ID in the output?

I think clustering should change too - thanks in advance.

**Comment by AroneyS** [comment]
> Thanks @AroneyS can you also add a comment line (i.e. starts with #) giving the %ID in the output?

Not sure what this means. Add a comment that describes the logging.info line?
e.g. `# Log % sequence ID to output`

**Comment by wwood** [comment]
```
# Clustered using 95% ANI
.... (rest of appraise output)
```

Sense it makes?

File: singlem/appraiser.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def _appraise_inexactly(self, metagenome_otu_table_collection,
 
         '''
         if sequence_identity:
+            logging.info("Appraising with %i sequence identity cutoff " % sequence_identity)
```
Comment (AroneyS) [inline singlem/appraiser.py:120]: @wwood The % cutoff used is logged here. Is this what you are after?

**Comment by wwood** [comment]
Was thinking of putting that in stdout, not in logging, so info is carried forward, yeh?

**Comment by AroneyS** [comment]
Like this?

**Comment by wwood** [comment]
Sounds good but use %f not i

-\-------------
Ben Woodcroft
Group leader, Centre for Microbiome Research, QUT
________________________________

**Comment by wwood** [comment]
Ta

**Pull request merged**